src/app/Library/CrudPanel/Traits/Columns.php

```diff
@@ -253,7 +253,7 @@ public function getColumnsRelationships()
         $columns = $this->columns();
 
         return collect($columns)->pluck('entity')->reject(function ($value, $key) {
-            return $value == null;
+            return $value == null || $value == false;
```

Comment:
@pxpm since we are not using `===` these two conditions evaluate to the same.
So if `$value` is `null`, or `false` `$value == null` is true, and `$value == false` is also true.

I think we may either check just for `!$value` (anything nullish gets rejected) or we use the type equal operator on both conditions `===`.

Reply:
Indeed. thanks

Reply:
@pxpm let me add another note here 😅

I think we should change this validation to `!$value`.

By having just `$value == null`, developers who read it don't realize if the validation is only rejecting `null` variables, or any falsy variable (`0`, `false`, `''`). By default one's may assume it's only rejecting `null`.

- When having `!$value` we know every falsy value is going to be rejected.
- When having `$value === null || $value === false` we know for sure `null`'s and `false`'s are going to be rejected, and nothing else.

I try to use the type-safe equality operators always, maybe, @pxpm and @tabacitu if you guys agree on this concern we may try to set scrutinizer to validate equality operators too (don't know if it's possible).

Reply:
Indeed @promatik I changed it to `!$value` already